udata/harvest/backends/base.py

```diff
@@ -127,92 +128,92 @@ def has_feature(self, key):
     def get_filters(self):
         return self.config.get('filters', [])
 
-    def harvest(self):
-        '''Start the harvesting process'''
-        if self.perform_initialization() is not None:
-            self.process_items()
-            self.finalize()
-        return self.job
 
-    def perform_initialization(self):
-        '''Initialize the harvesting for a given job'''
-        log.debug('Initializing backend')
+    """
```

Comment:
Maybe move this docstring at the top of the class?

Reply:
Done in b9b41e23f